demos/python_demos/asl_recognition_demo/README.md

```diff
@@ -0,0 +1,84 @@
+
+## Running
+
+Run the application with the `-h` option to see the following usage message:
+
+```
+usage: asl_recognition_demo.py [-h] -m_a ACTION_MODEL -m_d DETECTION_MODEL -i
+                               INPUT -c CLASS_MAP [-d DEVICE]
+                               [-l CPU_EXTENSION] [--no_show]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -m_a ACTION_MODEL, --action_model ACTION_MODEL
+                        Required. Path to an .xml file with a trained asl
+                        recognition model.
+  -m_d DETECTION_MODEL, --detection_model DETECTION_MODEL
+                        Required. Path to an .xml file with a trained person
+                        detector model.
+  -i INPUT, --input INPUT
+                        Required. Path to a video file or a device node of a
+                        web-camera.
+  -c CLASS_MAP, --class_map CLASS_MAP
+                        Required. Path to a file with ASL classes.
+  -d DEVICE, --device DEVICE
+                        Optional. Specify the target device to infer on: CPU,
+                        GPU, FPGA, HDDL or MYRIAD. The demo will look for a
+                        suitable plugin for device specified (by default, it
+                        is CPU).
+  -l CPU_EXTENSION, --cpu_extension CPU_EXTENSION
+                        Optional. Required for CPU custom layers. Absolute
+                        path to a shared library with the kernels
+                        implementations.
+  --no_show             Optional. Do not visualize inference results.
+```
+
+Running the application with an empty list of options yields the short version of the usage message and an error message.
+
+To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../../tools/downloader/README.md) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/).
+
+> **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (`*.xml` + `*.bin`) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
+
+To run the demo, please provide paths to the ASL recognition and person detection models in the IR format, to a file with class names, and to an input video:
+```bash
+python asl_recognition_demo.py \
+-m_a /home/user/asl-recognition-0003.xml \
+-m_d /home/user/person-detection-asl-0001.xml \
+-i /dev/video0 \
+-c ./classes.json
```

Comment:
Please make the arguments more platform-agnostic (you should be able to use `0` for the input, now that you've added that `int` conversion).

Reply:
Done.